docs/conf.py

```diff
@@ -24,7 +24,13 @@
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
-autodoc_mock_imports = ["adafruit_logging", "adafruit_jwt", "rtc"]
+autodoc_mock_imports = [
```

Comment:
You can remove these autodoc mock imports since the requirements should now be installed correctly by the CI/user.

Reply:
Sphinx still fails without the mock imports
![image](https://user-images.githubusercontent.com/23083317/176202438-17cb962d-a298-4bda-8b18-137774eedfc4.png)